ml-dhs-aws/src/main/java/com/marklogic/client/dhs/CSVConverter.java

```diff
@@ -11,52 +11,40 @@
 import java.util.Iterator;
 
 public class CSVConverter {
-   private CsvMapper csvMapper;
-//   private CsvFactory factory;
-
-   public CSVConverter() {
-      csvMapper = new CsvMapper()
-            .configure(CsvParser.Feature.ALLOW_TRAILING_COMMA, true)
-            .configure(CsvParser.Feature.FAIL_ON_MISSING_COLUMNS, false)
-            .configure(CsvParser.Feature.IGNORE_TRAILING_UNMAPPABLE, false)
-            .configure(CsvParser.Feature.INSERT_NULLS_FOR_MISSING_COLUMNS, false)
-            .configure(CsvParser.Feature.SKIP_EMPTY_LINES, true)
-            .configure(CsvParser.Feature.TRIM_SPACES, true)
-            .configure(CsvParser.Feature.WRAP_AS_ARRAY, false);
-/*
-      factory = new CsvFactory()
-            .configure(CsvParser.Feature.ALLOW_TRAILING_COMMA, true)
-            .configure(CsvParser.Feature.FAIL_ON_MISSING_COLUMNS, false)
-            .configure(CsvParser.Feature.IGNORE_TRAILING_UNMAPPABLE, false)
-            .configure(CsvParser.Feature.INSERT_NULLS_FOR_MISSING_COLUMNS, false)
-            .configure(CsvParser.Feature.SKIP_EMPTY_LINES, true)
-            .configure(CsvParser.Feature.TRIM_SPACES, true)
-            .configure(CsvParser.Feature.WRAP_AS_ARRAY, false);
-  */
-   }
+	private CsvMapper csvMapper;
+	private CountInputStream countInputStream;
 
-// TODO: in job control file, expose schema control over whether the value is a JSON string, number, or boolean
-   public Iterator<ObjectNode> convertObject(InputStream csvSource) throws IOException {
-      CsvSchema firstLineSchema = CsvSchema.emptySchema().withHeader();
+	public CountInputStream getCountInputStream() {
+		return countInputStream;
+	}
+
+	public void setCountInputStream(CountInputStream countInputStream) {
+		this.countInputStream = countInputStream;
+	}
```

Comment:
Do we need a public setter and getter?  Or could CSVConverter encapsulate the use of CountInputStream?